test/README.md

```diff
@@ -0,0 +1,247 @@
+# Test
```

Comment:
can we replace this with a README that explains how to run our tests? most of this doesn't apply to this repo